servicemanager/service/smplayservice.py

```diff
@@ -95,6 +97,10 @@ def start_from_binary(self):
         parent, _ = os.path.split(unzip_dir)
         force_pushdir(parent)
 
+        if "frontend" in self.service_data and self.service_data["frontend"]:
```

Comment:
could we rename this to be "is_frontend", you can also do "if self.service_data.get("is_frontend", False):"